### DISCUSSION TITLE
is this an error??

### DISCUSSION THREAD

**Discussion by hjonathan2019** [coinfoundry/miningcore#1469] [General]
hello to all, i have this in miningcore 

sending :{"jsonrpc":"2.0","method":"mining.set_difficult","param":{1],"id":null

follow with a lot of numbers... this is an error??? or its normal??

![image](https://user-images.githubusercontent.com/59586038/195862752-53c968c4-d599-4403-978c-90aeb24d3c72.png)

**Comment by oliverw** [comment]
You've turned on debug logging.

**Reply by hjonathan2019** [reply]
ok, but can you see in this screenshot any error??? 

and, another thing. Can you recommend diificulty settings for ergo pool?? i have almost 14 ghs and still not getting any block....

**Reply by hjonathan2019** [reply]
coming back to this topic... i need to know if this is a bug please
![image](https://user-images.githubusercontent.com/59586038/197309104-daff82fc-ef97-41ce-8dca-1b843da04326.png)

**Reply by jvxis** [reply]
No. It is not a bug. Expected behavior. When a new block is broadcasted through p2p network.😎

**Reply by MiningCryptoLive** [reply]
> ok, but can you see in this screenshot any error??? 
> 
> and, another thing. Can you recommend diificulty settings for ergo pool?? i have almost 14 ghs and still not getting any block....

14 ghs could take weeks to months to find a block depending on your luck
